src/index.html

```diff
@@ -11,6 +11,47 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="#"
+      method="post"
+      class="form form--big"
+      data-qa="big"
+    >
+      <img
+        src="images/Search.svg"
+        alt="search"
+        class="icon icon--big"
```

Comment:
```suggestion
      class="form"
      data-qa="big"
    >
      <img
        src="images/Search.svg"
        alt="search"
        class="icon"
```
no need modifiers for big form. set necessary styles in block class (form, icon) and change someproperties which needs in form--small, icon--small modifiers. same for other elements.